dingo.go

```diff
@@ -148,7 +148,14 @@ func (injector *Injector) InitModules(modules ...Module) {
 		known := make(map[string]*Binding)
 		for _, binding := range bindings {
 			if known, ok := known[binding.annotatedWith]; ok && !known.equal(binding) {
-				panic(fmt.Sprintf("already known binding for %s with annotation '%s' | Known binding: %#v%#v Try %#v%#v", typ, binding.annotatedWith, known.to.PkgPath(), known.to.Name(), binding.to.PkgPath(), binding.to.Name()))
+				var knownBinding, duplicateBinding string
+				if known.to != nil {
+					knownBinding = fmt.Sprintf("%#v%#v", known.to.PkgPath(), known.to.Name())
+				}
+				if binding.to != nil {
+					duplicateBinding = fmt.Sprintf("%#v%#v", binding.to.PkgPath(), binding.to.Name())
+				}
+				panic(fmt.Sprintf("already known binding for %s with annotation '%s' | Known binding: %s Try %s", typ, binding.annotatedWith, knownBinding, duplicateBinding))
```

Comment:
I would just suggest single quotes around every binding output to see that there is eventually an empty string. Looks strange otherwise.

Reply:
There is `%q` for that